### DISCUSSION TITLE
Release for alpine infrastructure ?

### DISCUSSION THREAD

**Discussion by Zylon04** [foundry-rs/foundry#8500] [Q&A] [answered]
Hello, 
I would like to use cast in docker on an Alpine infrastructure (in the CI of a project to be more precise). 
For now, i must clone the foundry repo and then compile the project instead of using the [precompiled binaries](https://github.com/foundry-rs/foundry/releases) because there are no binary for alpine as far as i know.
Would it be interesting for foundry to have a binary for alpine infrastructure ?

**Comment by zerosnacks** [comment, marked as answer]
Docker has recently been updated to include Alpine support: https://github.com/foundry-rs/foundry/blob/master/Dockerfile

See: https://github.com/foundry-rs/foundry/issues/8498
